waltz-ng/client/common/componets/entity-link/entity-link.js

```diff
@@ -17,16 +17,25 @@ const initialState = {
 };
 
 
-function controller() {
+function controller($state) {
     const vm = initialiseData(this, initialState);
 
     vm.$onChanges = (changes) => {
         if (vm.entityRef) {
             vm.viewState = kindToViewState(vm.entityRef.kind);
+            if (vm.viewState) {
```

Comment:
but yes, I could just use `viewUrl` in the ng-if now instead.. will make the change